### ISSUE TITLE
Selecting different learning rates for Discriminator and Generator

### ISSUE THREAD

**Issue by harjatinsingh** [phillipi/pix2pix#92]
I see that very early on in the training process, ErrG goes close to 10 whereas ErrD is close to 0.0001 and it doesn't seem to change from there onward. I wanted to decrease learning rate for discriminator to equalize the losses.

**Comment by phillipi** [comment]
You can do so by editing `optimStateG` and `optimStateD` in `train.lua`.
